getting_started/nxp/sof_imx_user_guide.rst

```diff
@@ -244,3 +244,102 @@ After, booting we will see now that SOF sound card will have two subdevices:
++---------------+--------------------+----------------------------------------------------+--------------------+
+
+As usual we will boot the i.MX8QM board using imx8qm-mek-sof-wm8960.dtb. We need to use sof-imx8-src-wm8960.tplg topology file.
+
+.. code-block:: bash
+
+   $ cp /lib/firmware/imx/sof-tplg/sof-imx8-src-wm8960-mixer.tplg /lib/firmware/imx/sof-tplg/sof-imx8-wm8960.tplg
+
+Here are several runs with aplay and various rates and formats.
+
+.. code-block:: bash
+
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S16_LE -c 2 -r 8000 -t raw /mnt/test/samples_16b/audio8k16b2c.wav
+   Playing raw data '/mnt/test/samples_16b/audio8k16b2c.wav' : Signed 16 bit Little Endian, Rate 8000 Hz, Stereo
+   
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S16_LE -c 2 -r 16000 -t raw /mnt/test/samples_16b/audio16k16b2c.wav
+   Playing raw data '/mnt/test/samples_16b/audio16k16b2c.wav' : Signed 16 bit Little Endian, Rate 16000 Hz, Stereo
+   
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S24_LE -c 2 -r 32000 -t raw /mnt/test/samples/audio32k24b2c.wav
+   Playing raw data '/mnt/test/samples/audio32k24b2c.wav' : Signed 24 bit Little Endian, Rate 32000 Hz, Stereo
+   
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S24_LE -c 2 -r 44100 -t raw /mnt/test/samples/audio44k24b2c.wav
+   Playing raw data '/mnt/test/samples/audio44k24b2c.wav' : Signed 24 bit Little Endian, Rate 44100 Hz, Stereo
+   
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S32_LE -c 2 -r 48000 -t raw /mnt/test/samples_32b/audio48k32b2c.wav
+   Playing raw data '/mnt/test/samples_32b/audio48k32b2c.wav' : Signed 32 bit Little Endian, Rate 48000 Hz, Stereo
+   
+   root@imx8qmmek:~# aplay -Dhw:1,0 -f S32_LE -c 2 -r 96000 -t raw /mnt/test/samples_32b/audio96k32b2c.wav
+   Playing raw data '/mnt/test/samples_32b/audio96k32b2c.wav' : Signed 32 bit Little Endian, Rate 96000 Hz, Stereo
+
+Compress audio
+--------------
+
+In order to use DSP to decode/encode compress audio we make use of `ALSA Compress Offload API <https://www.kernel.org/doc/html/latest/sound/designs/compress-offload.html>`_
+
+Supported codecs on i.MX8QM:
+
++---------------+--------------------+----------------+-------------------------------------------------+
+|   codec       |              topology               |           Test command                          |
++===============+=====================================+=================================================+
+|    PCM        | sof-imx8-processing-pcm-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I PCM sample.wav  |
++---------------+-------------------------------------+-------------------------------------------------+
+|    MP3        | sof-imx8-processing-mp3-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I MP3 sample.mp3  |
++---------------+-------------------------------------+-------------------------------------------------+
+|    AAC        | sof-imx8-processing-aac-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I MP3 sample.aac  |
```

Comment:
"-I MP3" Is this intended?

Reply:
@paulstelian97 thanks it's a typo. Will fix it with next batch of changes for documentation.